### PR TITLE
docs: edited README.md

### PR DESCRIPTION
**Pull request by sunnieeeYoon**
### Description of change
- edited img URL in README.md

### Pull-Request Checklist

- [x] Code is up-to-date with the `main` branch
- [ ] There are new or updated unit tests validating the change
- [ ] Documentation has been updated to reflect this change
- [x] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/JunbaeJs/junbae-mode/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div align="center">
   <a href="https://marketplace.visualstudio.com/items?itemName=JunbaeJs.junbae-mode&ssr=false#overview">
-    <img src="../junbae-mode/images/junbae_logo.png" height="128">
+    <img src="https://github.com/JunbaeJs/junbae-mode/blob/main/images/junbae_logo.png?raw=true" height="128">
```
Comment (changchanghwang) [inline README.md]: 1. Use absolute route path
2. modify package.json -> repository.url (remove git+, use https protocol)

File: README.md [hunk 2]
```diff
@@ -41,18 +42,18 @@ Come and immerse yourself in the adorable allure of Junbae together!
 - Those who want to be noticed at work!
 
 ## Warning
-
 Junbae doesn't directly help you with your development or improve your skills.<br/>
 Be careful not to get too caught up with Junbae and forget your tasks.
 
 But isn't it adorable?
 
+
```
Comment (changchanghwang) [inline README.md]: why 2 empty rows?

Comment (sunnieeeYoon) [reply README.md]: Seems like it changed automatically.

File: README.md [hunk 3]
```diff
@@ -33,6 +33,7 @@ Enchanted by Junbae's charm, we often find ourselves thinking of her even while
 That's why we've introduced the Junbae mode, allowing you to enjoy Junbae's presence during typing.<br/>
 Come and immerse yourself in the adorable allure of Junbae together!
 
+
```
Comment (changchanghwang) [inline README.md]: 2 lines

**Review by myunggyuns — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (myunggyuns) [reply README.md]: How about using this way to upload image?
reference link: https://mrgamza.tistory.com/778

Comment (changchanghwang) [reply README.md]: we already uploaded image to github repository. so we need just route

**Review by changchanghwang — APPROVED** [review]

**Pull request merged**